.github/workflows/build-k8s-container.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name: Build and Push Kubernetes Probe Docker image
+
+on:
+  push:
+    branches: [ 2.x ]
+    paths:
+      - '.github/workflows/build-k8s-container.yml'
+      - 'gratia-output-kapel/**'
+      - 'common/**'
+  repository_dispatch:
+    types:
+      - dispatch-build
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'opensciencegrid/')
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: make date tag
+      id: mkdatetag
+      run: echo "dtag=$(date +%Y%m%d-%H%M)" >> $GITHUB_OUTPUT
+
+    - name: make tags
+      id: mktags
+      env:
+        DTAG=${{ steps.mkdatetag.outputs.dtag }}
+      run: |
+        reponame="osg-htc/gratia-probe-k8s"
+        tags=()
+        for registry in docker.io hub.opensciencegrid.org; do
+            tags+=("$registry/$reponame:release")
```

Comment:
Since we pull things out of OSG 23, we should probably align tags, e.g. `23-release`